.circleci/config.yml

```diff
@@ -0,0 +1,26 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/wp-e2e-tests
+    docker:
+      - image: 'circleci/node:8.11.0-browsers'
+        environment:
+          NODE_ENV: test
+          TARGET: VISDIFF
+    steps:
+      - checkout
+      - run: echo 127.0.0.1 lorempixel.com | sudo tee -a /etc/hosts
```

Comment:
do we still need these overrides?

Reply:
No, these overrides were for issues when the visdiffs ran against devdocs: https://github.com/Automattic/wp-e2e-tests/pull/278

Since we're not running against devdocs anymore we can remove these.